x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.ts

```diff
@@ -67,44 +67,60 @@ export const callEnterpriseSearchConfigAPI = async ({
       publicUrl: stripTrailingSlash(data?.settings?.external_url),
       readOnlyMode: !!data?.settings?.read_only_mode,
       ilmEnabled: !!data?.settings?.ilm_enabled,
+      isFederatedAuth: !!data?.settings?.is_federated_auth, // i.e., not standard auth
```

Comment:
Brian moved this to a top-level setting, not just nested under a Workplace Search specific key - I agree with him in this case since it affects both AS & WS.

So just an FYI @scottybollinger  that the location of `isFederatedAuth` has moved, if that's okay